src/test/java/com/wikia/webdriver/testcases/desktop/discussions/PollsTests.java

```diff
@@ -12,23 +11,25 @@
 import com.wikia.webdriver.elements.communities.mobile.components.discussions.common.*;
 import com.wikia.webdriver.elements.communities.mobile.pages.discussions.PostDetailsPage;
 import com.wikia.webdriver.elements.communities.mobile.pages.discussions.PostsListPage;
+
 import org.testng.annotations.Test;
 
+@Execute(onWikia = "qadiscussions", language = "de")
 public class PollsTests extends NewTestTemplate {
 
     /* TODO: Use fixtures to create Poll post */
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)